tooling/templatize/pkg/pipeline/common.go

```diff
@@ -0,0 +1,47 @@
+package pipeline
```

Comment:
Please add tests for these

Reply:
Although @geoberle you could do a round-trip test e.g. `assert.Empty(cmp.diff(foo, foo.DeepCopy(foo.PipelinePath))`